spec/services/location_data_service_spec.rb

```diff
@@ -209,6 +211,10 @@
       expect(location_record9.remote_storage).to eq ''
       expect(location_record10.label).to eq ''
       expect(location_record11.label).to eq 'Scribner Library: Common Works Collection'
+      expect(location_record12.fulfillment_unit).to eq 'Reserves'
+      expect(location_record12.requestable).to eq false
+      expect(location_record13.fulfillment_unit).to eq 'Reserves'
+      expect(location_record13.requestable).to eq false
```

Comment:
I would suggest moving these 4 expects (lines 214-217) to their own `it` section since they test a very specific feature (that locations with fulfillment unit "Reserves" are not requestable) and it would be good to highlight this on the test.